packages/components/src/Combobox/components/ComboboxContent/ComboboxContentList/ComboboxContentList.css

```diff
@@ -35,17 +35,17 @@
   bottom: var(--space-small);
   background: linear-gradient(
     180deg,
-    rgba(255, 255, 255, 0) 0%,
-    rgb(255, 255, 255) 100%
+    transparent 0%,
+    var(--color-surface) 100%
   );
 }
 
 .container::before {
   top: var(--space-small);
   background: linear-gradient(
-    to bottom,
-    rgba(255, 255, 255, 1),
-    rgba(255, 255, 255, 0)
+    180deg,
```

Comment:
either syntax works but I just made the two gradients in the same file use the same format